pennylane/qnn/keras.py

```diff
@@ -196,7 +202,13 @@ def qnode(inputs, weights):
     """
 
     def __init__(
```

Comment:
> In the CodeFactor failed check, they seem to have an upper limit of 5 arguments for the constructor. Do you believe there is a good way to remove that argument or include it in the keyword arguments, to stay consistent with their code design?

@vbelis my addition `argnum` is already a keyword argument, are you suggesting leaving it for `kwargs`? If so that might clash with TF-Keras backend here. Of course, it can be removed from the `kwargs` dict etc but IMHO that would make unnecessarily dirty code so I prefer to leave such implementation to the main developers.

Reply:
[This](https://github.com/PennyLaneAI/pennylane/pull/2069#issuecomment-1048014745) is added after the `__init__` function. Thanks @josh146